### ISSUE TITLE
Can't use username to reset password

### ISSUE THREAD

**Issue by Madelharri** [marttiphpbb/phpbb-ext-usernameoremailpasswordreset#1]
I've enabled the extension but when I try to reset my password using my username I get the message that it doesn't include an "@", so it obviously is not working.  I've tried disabling other extensions to see if any of them are causing a conflict but still cannot get it to work.  I'm using PHPBB 3.2.5 and PHP 7.0.33.  Any ideas what is wrong?

**Comment by marttiphpbb** [comment]
No, from phpBB 3.2.5 the standard way is email-only reset, no more username. That's why this extension doesn't work anymore. The username field is missing. Probably this extension is going to be discontinued.

**Comment by Madelharri** [comment]
Thanks for letting me know.

 

Madeleine McMullin 

Source Marketing

Tel : +33 (0)4  30 44 50 25, +44 (0)1797 208021 (Rings in France)

 <http://www.marketing66.fr/> www.sourcemarketing.net

Siège/Registered office : 2 bis rue de la Tour, 66110 Amélie Les Bains Palalda

 

Si vous avez reçu ce message en erreur, veuillez le supprimer et me notifier s’il vous plait.

If you’ve received this message in error, please delete it and let me know.
